Doc/library/sqlite3.rst

```diff
@@ -2269,8 +2339,67 @@ Explanation
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+.. _sqlite3-transaction-control-autocommit:
+
+Transaction control via the ``autocommit`` attribute
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The recommended way of controlling transaction behaviour is through
+the :attr:`Connection.autocommit` attribute.
```

Comment:
```suggestion
the :attr:`Connection.autocommit`.
```

Avoid repetition in the next sentence and sounding too contradictory with saying that setting it as an _attribute_ rather than though the parameter is _not_ recommended.